pkgs/tools/security/ioccheck/default.nix

```diff
@@ -49,6 +46,15 @@ buildPythonApplication rec {
 
   nativeBuildInputs = with py.pkgs; [
     poetry-core
+    pythonRelaxDepsHook
+  ];
+
+  pythonRelaxDeps = [
```

Comment:
Again is there any reason why these dependencies are relaxed? Is it because upstream they are not up to date ?

Reply:
They were also relaxed before, just in `postPatch`. I'm only adding pyfiglet which previously caused the build to fail.